awive/config.py

```diff
@@ -95,7 +96,45 @@ def di(i: int, j: int):
         x[:, 0] *= -1
         return x.tolist()
 
+    def convert_str_keys_to_tuples(
+        self, input_dict: dict[str, float]
+    ) -> dict[tuple[int, int], float]:
+        """Convert string-represented tuple keys to actual tuples"""
```

Comment:
Add google style docstring. include argument description without types.
```suggestion
        """Convert string-represented tuple keys to actual tuples."""
```